### PR TITLE
Welcome message

### PR DESCRIPTION
**Pull request by KellerFuchs**
- [x] Makes the welcome banner easier to edit and share between various shells.
- [x] Fixup the banner in various minor ways
- [x] Make the banner display under ZSH (#30)

### CODE REVIEW

**Comment by KellerFuchs** [comment]
Should we rather put the message in `/etc` ?

Pro: we can edit it without relying on users updating dotfiles.
Con: deployment to `to1` isn't currently possible.

**Comment by daurnimator** [comment]
Looks reasonable.

**Comment by KellerFuchs** [comment]
@daurnimator OK, making a PR before bed.

File: bash/.bash_profile [hunk 1]
```diff
@@ -1,47 +1,8 @@
...
-echo "   * To learn more about us and our offerings type: man hashbang";
-echo " ";
-echo " Like what we're doing? Consider donating to expand our efforts.";
-echo "  * Bitcoin - [ 1DtTvCLiUMhs21QcETQzLyiqxoopUjqBSU ]";
-echo "  * Google Wallet - [ donate@hashbang.sh ]";
-echo "  * PayPal - [ http://goo.gl/aSQWy0 ]";
-echo " ";
-echo " Community shell servers generously sponsored by: (http://atlantic.net)";
+# Source the greetings banner
+[ ! -d "/etc/hashbang" ] || /etc/hashbang/welcome
```
Comment (KellerFuchs) [inline bash/.bash_profile]: - `[ ! a ] || b` sets the correct error code.
- It is overall more readable than `[ a ] && b`.

**Pull request merged**